utils/cleanupResources.ts

```diff
@@ -9,21 +11,27 @@ for (const envVar of ['PINECONE_API_KEY']) {
   }
 }
 
-var pinecone = require('../dist');
-
 (async () => {
   const p = new pinecone.Pinecone();
 
   const collectionList = await p.listCollections();
-  for (const collection of collectionList.collections) {
-    console.log(`Deleting collection ${collection.name}`);
-    await p.deleteCollection(collection.name);
+  if (collectionList.collections) {
+    for (const collection of collectionList.collections) {
+      console.log(`Deleting collection ${collection.name}`);
+      await p.deleteCollection(collection.name);
+    }
+  } else {
+    console.log('No collections found to delete');
```

Comment:
Added this if stmt (and the one below) b/c we were getting a TS error, since `collectionList.collections` could technically be undefined.

Reply:
(We were likely _not_ getting these type errors when using CommonJS, since Typescript apparently interprets module boundaries and types more loosely in CommonJS vs ESModules. See [here](https://github.com/pinecone-io/pinecone-ts-client/pull/252/files#r1725714777), too.)